### DISCUSSION TITLE
Scene transition not working with some bulbs

### DISCUSSION THREAD

**Discussion by snueffel88** [Koenkk/zigbee2mqtt#6812] [Devices]
Hey, 

I just tried to setup some scenes but I'm experiencing problems with some bulbs while a transition time is set.

Storing and recalling scenes works quite well using `scene_add `and `scene_recall ` as described in the tutorial. If I set a transition time, it works for Philips Hue bulbs quite well, too.

However, I tried this with a "Müller Licht Tint LED bulb GU10" and here the transition shows a strange behavior. The bulb waits for the given time without any change and changes its color spontaneously afterwards. So the transition time seems to be accepted in general, but there is no smooth transition.

I'm not very experienced how scenes work, so first I thought this could be a problem of the bulbs. For this reason I borrowed a Ledvance GU10 bulb and one from Hornbach (FLAIR Viyu, looks similar to the Mueller Licht) - and both showed the same behavior. 

For this reason I am wondering if this really is a problem of the bulbs or if this is something which can be solved at the coordinator. Does someone have an idea or can at least confirm this observation? I appreciate to support with logs if needed.

Best regards

**Comment by baku104788** [comment]
Any news on this ? I'm on 2.6.1 using philips hue bulbs and it seems the transition no longer works.  No matter what I do it is using a transition of about 3 seconds.
